### ISSUE TITLE
Search only working for Next, no results for versioned documents

### ISSUE THREAD

**Issue by marcusnewton** [cmfcmf/docusaurus-search-local#63]
### Bug description 
I'm only getting search results when I navigate to `/next`. When using search for other versions (including latest), I do not see any search results. This is despite `search-index.json` seemingly indexing all versions correctly.

### Reproduction
Docusaurus configuration [here](https://github.com/beehive-innovation/rain-protocol/tree/develop/docusaurus)
Build output [here](https://github.com/beehive-innovation/rain-protocol/tree/gh-pages)
Site at https://beehive-innovation.github.io/rain-protocol/

#### Package Versions
```
docusaurus@0.0.0 /home/marcus/rain-protocol/docusaurus
├── @cmfcmf/docusaurus-search-local@0.6.6 
├── @docusaurus/core@2.0.0-beta.5 
└─┬ @docusaurus/preset-classic@2.0.0-beta.5
  ├── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/plugin-content-blog@2.0.0-beta.5
  │ ├── @docusaurus/core@2.0.0-beta.5  deduped
  │ └─┬ @docusaurus/mdx-loader@2.0.0-beta.5
  │   └── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/plugin-content-docs@2.0.0-beta.5
  │ └── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/plugin-content-pages@2.0.0-beta.5
  │ └── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/plugin-debug@2.0.0-beta.5
  │ └── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/plugin-google-analytics@2.0.0-beta.5
  │ └── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/plugin-google-gtag@2.0.0-beta.5
  │ └── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/plugin-sitemap@2.0.0-beta.5
  │ └── @docusaurus/core@2.0.0-beta.5  deduped
  ├─┬ @docusaurus/theme-classic@2.0.0-beta.5
  │ ├── @docusaurus/core@2.0.0-beta.5  deduped
  │ └─┬ @docusaurus/theme-common@2.0.0-beta.5
  │   └── @docusaurus/core@2.0.0-beta.5  deduped
  └─┬ @docusaurus/theme-search-algolia@2.0.0-beta.5
    └── @docusaurus/core@2.0.0-beta.5  deduped
```

#### Node Version
v14.18.0

#### Docusaurus Environment

- Are you using a custom theme?
No
- Paste the plugin config of the local search plugin from your `docusaurus.config.js` file
```
plugins: [
    [
      require.resolve("@cmfcmf/docusaurus-search-local"),
      {
        docsRouteBasePath: "/",
      },
    ],
  ],
```

### Additional context
Working `docs/` folder is placed a level above `docusaurus/` folder, which is not typical. 
I am also in docs-only mode.

I noticed this `Skipping search index generation` message when building, but it seems to happily index everything correctly after.
```
[en] Creating an optimized production build...
[Local Search] [INFO]: Skipping search index generation for documentation because directory /home/marcus/rain-protocol/docusaurus/docs does not exist.

✔ Client
  

✔ Server
  Compiled successfully in 6.70s


✔ Client
  

● Server █████████████████████████ cache (99%) shutdown IdleFileCachePlugin
 stored

[Local Search] [INFO]: Gathering documents
[Local Search] [INFO]: Parsing documents
[Local Search] [INFO]: Building index
[Local Search] [INFO]: Writing index to disk
[Local Search] [INFO]: Index written to disk, success!
Success! Generated static files in "build".

Use `npm run serve` command to test your build locally.
```

**Comment by cmfcmf** [comment]
Hey @marcusnewton - sorry for the delay. As you guessed, this is caused by the docs folder not being in the usual location. I have merged a PR (#78) that adds a new `docsPath` config option which allows you to specify the location of your docs folder and should fix your problem. I'll release a new version with this fix this weekend.

**Comment by marcusnewton** [comment]
Brilliant, glad you could fix it

**Comment by cmfcmf** [comment]
Should be fixed in 0.7.0.
